src/pudl/extract/eia176.py

```diff
@@ -0,0 +1,47 @@
+"""Extract EIA Form 176 data from CSVs.
+
+The EIA Form 176 archive also contains CSVs for EIA Form 191 and EIA Form 757.
```

Comment:
There's one source for the three forms, so I've coupled processing here.

Reply:
As in, the same ZIP file contains all three forms, but 191/757 aren't being extracted here, yet? That seems fine to me - do you think the empty 191/757 extractor files are useful from a documentation standpoint?

Reply:
I was planning to call the 191 and 757 extractors from within the 176 extractor module, to mirror and communicate the coupling in the data source (unless we decide to start grouping and naming pipelines in a way that's abstracted from specific EIA forms). I'll introduce unimplemented extractors in my next iteration at the very least, and the calls to them, so the structure and intention is better communicated without just adding noise with empty files.

Reply:
I ended up remove these stub files for EIA forms 191 and 757. They won't need to be coupled in the way I thought and they're only adding noise at this point.